README.md

```diff
@@ -46,29 +46,29 @@ We love the NYC 311 service and the open data products that come with it. In thi
 
 ### Task 1: Data Download
 
-Write a python script/notebook to download all service request records created in the **last week** (7 days) and has **HPD** as the responding agency, and store the data in a csv named `raw.csv` in a folder called `data`.
+Write a script to download all service request records created in the **last week** (7 days) and has **HPD** as the responding agency, and store the data in a csv named `raw.csv` in a folder called `data`.
```

Comment:
The idea behind this change is to make the download step more agnostic? If so, I think the omission of python works well for this - is it worth adding in more specificity in our directions or do we think that the "ambiguity" is a good window into how the candidate thinks/problem solves?